electron/main.ts

```diff
@@ -342,12 +342,15 @@ ipcMain.on('getLog', (event, appName) =>
 )
 
 ipcMain.on('removeFolder', async (e, [path, folderName]) => {
+  console.log({path});
```

Comment:
Maybe also logInfo instead console.log ?

Reply:
Is path a string ? I find this log really unreadable. What should tell me the log?
i would expect something like:
```typescript
const folderToDelete = path;
if (path === 'default') {
{
    const { defaultInstallPath } = await GlobalConfig.get().getSettings()
    const path = defaultInstallPath.replaceAll("'", '')
    folderToDelete = `${path}/${folderName}`
}
logInfo(`Removed folder ${folderToDelete}`)
```

Reply:
i removed the log, check the updated files because you're commenting on old commits